src/ExpandableListItem/ExpandableListItem.less

```diff
@@ -3,7 +3,10 @@
 	// Header
 	.moon-labeledtextitem-header {
 		.padding-start-end(0, @moon-spotlight-outset + 30px);
-		margin-bottom: 0px;
+
+		&.with-text {
+			margin-bottom: 0;
+		}
 
 		&:after {
 			position: absolute;
```

Comment:
I did not change the line below: `top: @moon-spotlight-outset - 12px;` since the 12px used to here doesn't refer to the spotlight-outset, but rather the vertical offset of the icon/glyph, which may be different from spotlight-outset.